### ISSUE TITLE
Command line with --save option does not save the final data in a project

### ISSUE THREAD

**Issue by petebankhead** [qupath/qupath#617]
**Describe the bug**
As in the title... see https://forum.image.sc/t/command-line-cannot-invoke-method-getimageplane/44127/16

**To Reproduce**
1. Create a script that adds objects to an image
2. Run the script from the command line using `-s`, `-p` and `-i` arguments
3. Check results - data is written without the new objects

**Expected behavior**
The data at the end of the script is saved.

**Additional context**
Fix looks straightforward, see https://github.com/qupath/qupath/blob/master/src/main/java/qupath/QuPath.java#L295 
Use instead
```java
Object result = runScript(project, imageData);
```

**Comment by petebankhead** [comment]
As a workaround, you can end script with
```groovy
getProjectEntry().saveImageData(getCurrentImageData())
```
and do **not** use the `--save` or `-s` option.

**Comment by petebankhead** [comment]
Fixed by https://github.com/qupath/qupath/pull/634 - the fix will be included in v0.3.0.
